plugins/test/runner.proto

```diff
@@ -49,7 +51,9 @@ message Input {
   bytes content = 2;
 
   // Serialized HTTP content, but provided via a separate file.
-  // File path can be absolute or relative to the test config file.
+  // For body input, don't include any content other than the body you would
```

Comment:
Doesn't this comment apply to the `content` field as well? If so, consider moving it to the top-level doc comment for the `Input` message. Or just remove it, I would have assumed that the file only contains body content, when used for body input.

Reply:
This is to differentiate from headers, where the file input is more than just the header. The file input includes the entire request.

Reply:
ok, SG